### DISCUSSION TITLE
Cloud preferences: wording

### DISCUSSION THREAD

**Discussion by DavidBailes** [audacity/audacity#6090] [Development]
There's currently a check box: "Show 'How would you like to Save?' modal"

'dialog' rather than 'modal' would be much more comprehensible to most users.

**Comment by petersampsonaudacity** [comment]
@LWinterberg @Tantacrul 

I _strongly_ agree with @DavidBailes 

"modal" means absolutely nothing to most non-programmers.

**Comment by petersampsonaudacity** [comment]
@LWinterberg @Tantacrul 

And "**On save behavior**" may be better as just "**Save behavior**"

Or even  "**Save dialog**"

Either would be better English

**Comment by Tantacrul** [comment]
Agree.

**Comment by petersampsonaudacity** [comment]
@Tantacrul   so do we need to add this as an issue ?

**Comment by Tantacrul** [comment]
Yes. @LWinterberg can you make the change from 'modal' to 'dialog' when you're back in action tomorrow?

**Reply by petersampsonaudacity** [reply]
I think you mean from modal to dialog

**Reply by Tantacrul** [reply]
I stand by what I said!

Oh wait, you're right :)

(edited the original comment)

**Comment by petersampsonaudacity** [comment]
Logged as:

 **Wording in the new Cloud Preferences #6095**
